docker-compose.yml

```diff
@@ -35,11 +35,17 @@ services:
       - POSTGRES_DB=resspectdb
       - POSTGRES_DATA_DIR=/docker-entrypoint-initdb.d
     volumes:
-      - ./resspectdb.sql:/docker-entrypoint-initdb.d/resspectdb.sql
       - type: bind
```

Comment:
I have come to having some questions about whether this `resspectdb` container is even necessary.

I cannot find any situation in which it is used.